### ISSUE TITLE
Class 'OneLogin_Saml2_Settings' not found .. metadata.php

### ISSUE THREAD

**Issue by vladsf** [onelogin/php-saml#371]
Hi!

demo1/ is broken.

**Comment by pitbulk** [comment]
```
Class 'OneLogin_Saml2_Settings' not found
```
means that the metadatap.php file was not able to load the OneLogin_Saml2_Settings class.

Are you using the php-saml 2.X version?
Otherwise, if you using php-saml 3.X you may use this version of [metadata.php](https://github.com/onelogin/php-saml/blob/3.1.1/demo1/metadata.php) that uses namespaces
```
use OneLogin\Saml2\Settings;
```

**Comment by vladsf** [comment]
I am using 'master' branch.

**Comment by pitbulk** [comment]
master is 2.X branch.

Are you loading the toolkit libs properly?

**Comment by vladsf** [comment]
I fetched latest release and found no `demo1/` directory so I made `git clone`. This might explain the problem.

**Comment by pitbulk** [comment]
The zip does not include demo code, just the toolkit itself, if you need the demo, then you need the git repository, but I guess you modified something so the demo1 is not able to load the libraries.

**Comment by vladsf** [comment]
Latest release available for download here is v3.1.0. Master branch is is 2.X. 
Surely I shot myself in the foot. I did not expect the master branch to be a major version behind.

**Comment by pitbulk** [comment]
3.X is incompatible with 2.X 

master is still set to 2.X to prevent environments using master branch in its composer to have their env broken.
